app/helpers/utils.rb

```diff
@@ -26,16 +26,18 @@ def self.mail_notice(input_mails, parsed_mails, newly_added)
     valid = parsed_mails.count
     invalid = total - valid
     already_present = (parsed_mails - newly_added).count
+    was = I18n.t("helpers.utils.was")
+    were = I18n.t("helpers.utils.were")
```

Comment:
Use I18n interpolation as "was", "were" etc are not directly translatable to many languages